runtime/jcl/common/jclexception.c

```diff
@@ -146,24 +144,28 @@ getStackTraceIterator(J9VMThread * vmThread, void * voidUserData, UDATA bytecode
 			}
 
 			/* Fill in module name and version */
-			if (j2seVersion >= J2SE_V11) {
+			if (JAVA_SPEC_VERSION >= 11) {
 				J9Module *module = NULL;
-				if (classLoader != NULL) {
+				if (NULL != classLoader) {
 					j9object_t classLoaderName = J9VMJAVALANGCLASSLOADER_CLASSLOADERNAME(vmThread, classLoader->classLoaderObject);
 					J9VMJAVALANGSTACKTRACEELEMENT_SET_CLASSLOADERNAME(vmThread, element, classLoaderName);
 				}
 				if (J9_ARE_NO_BITS_SET(vm->runtimeFlags, J9_RUNTIME_JAVA_BASE_MODULE_CREATED)) {
+					PORT_ACCESS_FROM_JAVAVM(vm);
+					char javaModuleVersion[4] = {0};
+					UDATA verLength = j9str_printf(PORTLIB, javaModuleVersion, 4, "%u", JAVA_SPEC_VERSION);
```

Comment:
This can be done via a macro, see https://github.com/eclipse-openj9/openj9/blob/master/runtime/vm/vmprops.c#L578-L580

Reply:
Perhaps we should move `J9_STR` to `j9.h`

Reply:
Moved to `j9comp.h`.